nautobot/extras/templates/extras/job_list.html

```diff
@@ -85,6 +86,7 @@ <h1>{% block title %}Jobs{% endblock %}</h1>
                                     {% endif %}
                                 </td>
                             </tr>
+                            {% endif %}
```

Comment:
Placement here seems wrong to me - shouldn't we also hide the result stats (below) for a hidden job?